### PR TITLE
[OBSDEF-8255] Add logging injecting labels for Fedsvc and IAM Atlas pods

### PR DESCRIPTION
**Pull request by voa808**
## Purpose
[OBSDEF-8255] Add labels for Fedsvc and IAM Atlas pods

## PR checklist
- [ ] make test passed
- [x] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [ ] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
https://asd-ecs-jenkins.isus.emc.com/jenkins/job/charts-custom-ci/210/

_Paste the output of your helm install/vSphere7 deployment testing here_
https://jira.cec.lab.emc.com/browse/OBSDEF-8255?focusedCommentId=6144492&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-6144492

### CODE REVIEW

**Review by sumithreddy — APPROVED** [review]
Labels LGTM. Please update the checklist

**Comment by sumithreddy** [comment]
> Labels LGTM. Please update the checklist

I merged component lables on these atlas deployment files https://github.com/EMCECS/charts/pull/848

**Comment by voa808** [comment]
@sumithreddy ohk, in that case, we can reject this PR and close it.

File: federation/templates/atlas_v1beta1_fedcluster.yaml [hunk 1]
```diff
@@ -24,6 +24,8 @@ spec:
           storage: {{ required "atlas.persistence.size is required" .Values.atlas.persistence.size }}
   labels:
 {{ include "common-lib.labels-sp-integrated-all" . | indent 4}}
+{{ include "common-monitoring-lib.logging-inject-labels" . | indent 4 }}
```
Comment (sumithreddy) [inline federation/templates/atlas_v1beta1_fedcluster.yaml:27]: common-monitoring-lib chart dependencies should be added.

File: objectscale-iam/templates/atlas_v1beta1_iamcluster.yaml [hunk 2]
```diff
@@ -22,6 +22,8 @@ spec:
           storage: {{ required "atlas.persistence.size is required" .Values.atlas.persistence.size }}
   labels:
 {{ include "common-lib.labels-sp-integrated-all" . | indent 4}}
+{{ include "common-monitoring-lib.logging-inject-labels" . | indent 4 }}
```
Comment (sumithreddy) [inline objectscale-iam/templates/atlas_v1beta1_iamcluster.yaml:25]: common-monitoring-lib chart dependencies should be added.

**Review by sumithreddy — CHANGES_REQUESTED** [review]
add dependent monitoring charts

File: federation/templates/atlas_v1beta1_fedcluster.yaml (hunk 1 above)
Comment (voa808) [reply federation/templates/atlas_v1beta1_fedcluster.yaml:28]: Done, thanks!

**Review by sumithreddy — APPROVED** [review]

**Review by SeemaTahaliyani — APPROVED** [review]

**Pull request merged**